src/unix/futex.c

```diff
@@ -1,10 +1,5 @@
 #include <unix_internal.h>
 
-struct futex {
```

Comment:
Any reason for making this public?

Reply:
Ah, that was from an earlier version of (global) futex_wake which took a pointer to the struct as an argument. Will fix, thanks.